libraries/chain/proposal_evaluator.cpp

```diff
@@ -103,6 +103,11 @@ object_id_type proposal_create_evaluator::do_apply(const proposal_create_operati
       std::set_difference(required_active.begin(), required_active.end(),
                           proposal.required_owner_approvals.begin(), proposal.required_owner_approvals.end(),
                           std::inserter(proposal.required_active_approvals, proposal.required_active_approvals.begin()));
+      if ( proposal.is_authorized_to_execute(d, proposal.fail_reason)
+            && proposal.review_period_time && proposal.fail_reason.empty() )
+      {
+         proposal.fail_reason = "Awaiting expiration of review period";
```

Comment:
``proposal`` is a non-const reference passed in to to the lambda. But once again, this will probably be backed out because of the misunderstanding of #730.